app/Repositories/User/UserRepository.php

```diff
@@ -24,12 +24,22 @@ public function getModel()
      */
     public function uploadAvatar($request, $name, $oldAvatarName, $pathUpload)
     {
-        $pathFile = $oldAvatarName;
-
         if (Storage::disk('local')->exists($oldAvatarName) && $oldAvatarName != config('settings.avatar_default_path')) {
             Storage::disk('local')->delete($oldAvatarName);
         }
 
         return $this->uploadUserAvatar($request, $name, $pathUpload);
     }
+
+    /**
+     * @param $keyword
+     * @return $this
```

Comment:
return $this là gì ạ, viết rõ doc

Reply:
return $this->model->where('name', 'LIKE', '%' . $keyword . '%')
            ->orWhere('email', 'LIKE', '%' . $keyword . '%');
được ko ạ

Reply:
da fix,
cái câu query trên chạy đc ak!